mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -490,6 +490,10 @@ If you were relying on accessing an individual flag by index (for example, `goog
 
 `google_service_networking_connection` now uses the Create endpoint instead of the Patch endpoint during the creation step. Previously, Patch was used as a workaround for an issue that has since been resolved.
 
+### `deleteConnection` method is used to delete the resource
```

Comment:
So previously, Terraform would not delete the resource, just remove the peering+abandoning? Should we change this to be more specific? Like 
```suggestion
### "terraform destroy" now fully deletes the resource instead of abandoning
```

Reply:
Looking at the code, I see we actually do abandon if the delete fails, so this suggestion is not valid.

Reply:
```suggestion
### `deleteConnection` API method is attempted to delete the resource
```
maybe?

Reply:
> Looking at the code, I see we actually do abandon if the delete fails, so this suggestion is not valid.

I am not sure if you checked the latest version of PR 8904. It does not abandon the resource after deletion failed and just logged the error. 

I think about it again. It looks like I should just returns the error instead of catch it and logged it? But I am not sure if it will affect the users or not to return the error.

Reply:
I misread and thought it was abandon, thanks for the correction. In the PR now, what happens after `terraform destroy` in that scenario? Just logs and keeps the state the same?

The previous functionality was to remove peering and then abandon the resource without deleting right?

Reply:
That is the good point.

Right, the previous implementation calls `removePeering` method. It removed peering, then abandoned the resource without deleting. The connection is not deleted actually, which caused the issue that the tenant projects/resources reach to the limit. https://buganizer.corp.google.com/issues/248298764

In the PR 8904, I change it to call the method `deleteConnection` and catch and log the error. After `terraform destroy`, the connection disappears in the state. That is not expected. 

I tested the case to not catch and return error if `deleteConnection` fails. `terraform destroy` fails with the error and the connection stays in the state. It looks like I should go with this way.